core/src/main/java/nl/nn/adapterframework/jms/JMSFacade.java

```diff
@@ -139,9 +140,9 @@ private AcknowledgeMode(int acknowledgeMode, String shortName) {
 		private DeliveryMode(int deliveryMode) {
 			this.deliveryMode = deliveryMode;
 		}
-		
+
 		public static DeliveryMode parse(int deliveryMode) {
-			return Misc.parseFromField(DeliveryMode.class, deliveryMode, d -> d.getDeliveryMode());
+			return EnumUtils.parseFromField(DeliveryMode.class, "DeliveryMode", deliveryMode, d -> d.getDeliveryMode());
```

Comment:
Ik zou verwachten dat DeliveryMode gedocumenteerd zou zijn.

Reply:
Dit is een internal use only piefje